src/verifier.cc

```diff
@@ -0,0 +1,124 @@
+// Copyright 2018 The Amber Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "src/verifier.h"
+
+#include <cmath>
+
+namespace amber {
+namespace {
+
+const float kEpsilon = 0.002f;
+
+bool IsFloatPixelEqualInt(float pixel, uint8_t expected) {
+  // TODO(jaebaek): Change kEpsilon to tolerance.
+  return std::fabs(pixel - static_cast<float>(expected) / 255.0f) < kEpsilon;
```

Comment:
The 255 here is very specific to the assumed framebuffer format.  

As a first step I'd make it a declared constant (thus giving it a name).   

This also assumes pixel values are 0-based.   
Again, this is a dependency on the assumed target format.
There are signed integer format, e.g. VK_FORMAT_R8G8B8A8_SINT.  So I'd add a TODO to generalze this.

For now, make a named constant for the 255.0

Reply:
This was probably there before, so I won't block this MR for this issue.